### DISCUSSION TITLE
My benefits have not arrived yet

### DISCUSSION THREAD

**Discussion by Leanderson01** [community/community#133546] [GitHub Education]
### Hi everyone,

I was approved on Monday, but I am still not allowed to use Copilot, for example.

**Comment by davevad93** [comment]
Hi @Leanderson01 ,

What's your application status?

- `Congrats! Your request for @Leanderson01 was approved and your benefits will be available soon.`

Or
 
- `Congrats! Your request for @Leanderson01 was approved and your benefits are now available.` 

If it's the first one, you'll need to wait until your benefits are available, which may take even a couple of weeks during periods when there's a high volume of applications. If it's the latter, you should have full access to your benefits, in that case I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education), select `I'm a verified student and I'm having trouble redeeming my Global Campus offers.` and explain your issue.
